sparklightautoml/transformers/scala_wrappers/is_holiday_transformer.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from typing import Dict, Set, List
+from uuid import uuid4
+
+from pyspark.ml.common import inherit_doc
+from pyspark.ml.param.shared import HasInputCols, HasOutputCols
+from pyspark.ml.util import JavaMLWritable
+from pyspark.ml.wrapper import JavaTransformer, JavaParams
+
+from sparklightautoml.mlwriters import CommonJavaToPythonMLReadable
+
+logger = logging.getLogger(__name__)
+
+
+@inherit_doc
+class IsHolidayTransformer(JavaTransformer, HasInputCols, HasOutputCols, CommonJavaToPythonMLReadable, JavaMLWritable):
+    """
+    Scala-based implementation of Target Encoder transformer
```

Comment:
does this comment correspond to actual use of the class?

Reply:
fixed